app/views/admin_request/show.html.erb

```diff
@@ -375,7 +375,7 @@
       </div>
     </div>
   <% end %>
-  <% if @info_request.incoming_messages.count > 0 %>
+  <% if @info_request.incoming_messages_count > 0 %>
```

Comment:
Pretty sure `.count` uses the counter cache value?

Reply:
I don't believe so:
```
[1] pry(main)> request = InfoRequest.first; p request.incoming_messages_count; p request.incoming_messages.count
  InfoRequest Load (8.0ms)  SELECT  "info_requests".* FROM "info_requests"  ORDER BY "info_requests"."id" ASC LIMIT 1
1
   (0.4ms)  SELECT COUNT(*) FROM "incoming_messages" WHERE "incoming_messages"."info_request_id" = $1  [["info_request_id", 1]]
1
```

Reply:
Oh, lame.

Its `#size` that picks the cached column or falls back to `count` if it doesn't exist:

> Tip: Although #count sounds like the more intuitive choice for counting the number of responses, this example uses #size, as #count will always do a COUNT query, while #size will skip the query if the responses are already loaded.
>
> – https://blog.appsignal.com/2018/06/19/activerecords-counter-cache.html